kubernetes_deployment.tf

```diff
@@ -11,7 +11,7 @@ resource "kubernetes_deployment" "tfc_agent" {
   }
 
   spec {
-    replicas = 1
+    replicas = var.agent_replicacount
```

Comment:
should this be `agent_replicas` to match the manifest attribute name?

Reply:
yes that might be better i've changed it